frontend/src/metabase/embedding/embedding-hub/utils/embedding-hub-steps.ts

```diff
@@ -84,6 +84,10 @@ export const getEmbeddingHubSteps = (): EmbeddingHubStep[] => {
       srcSet: "app/assets/img/embedding_hub_row_column_security@2x.png 2x",
       alt: t`Screenshot of configuring row and column security`,
     },
+    infoAlert: {
+      type: "always",
+      message: t`Log in using SSO to create a user with attributes before setting up \n row and column security.`,
```

Comment:
The `\n` makes the message breaks more natural.

<img width="1030" height="1040" alt="image" src="https://github.com/user-attachments/assets/721bab66-22e4-439f-a6cc-0555dc927158" />

Reply:
@npretto Good point, I didn't think about that. What about if we just add a `<br>` and add a context `{0} is a line break`?